lib/src/test/java/com/knuddels/jtokkit/reference/Cl100kLargeTokenizerTest.java

```diff
@@ -0,0 +1,30 @@
+package com.knuddels.jtokkit.reference;
+
+import com.knuddels.jtokkit.Encodings;
+import com.knuddels.jtokkit.api.Encoding;
+import com.knuddels.jtokkit.api.EncodingType;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+import static com.knuddels.jtokkit.TokenEncoder.VERY_LARGE_TOKENIZER_BYTE_THRESHOLD_KEY;
+
+class Cl100kLargeTokenizerTest extends Cl100kBaseTest {
```

Comment:
checked it via test coverage, the first only runs the fast array one, this one only runs the map based one